Cargo.toml

```diff
@@ -12,6 +12,7 @@ keywords = ["loadtesting", "performance", "web", "framework", "tool"]
 license = "Apache-2.0"
 
 [dependencies]
+async-std = { version = "1.6", features = ["unstable", "tokio02"] }
```

Comment:
I believe Tokio also [has bounded channels](https://github.com/tokio-rs/tokio/blob/master/tokio/src/sync/mpsc/bounded.rs). As it is already a dependency, is there any need to bring async-std ?

Reply:
The patch has now been reworked to use an `mpsc channel` implementing a modified leaky bucket algorithm, removing the extra dependency on `async-std`.